clients/java-logger/grpc-test-app/pom.xml

```diff
@@ -136,18 +136,18 @@
     <dependency>
       <groupId>org.springframework.boot</groupId>
       <artifactId>spring-boot-autoconfigure</artifactId>
-      <version>2.7.3</version>
+      <version>3.0.1</version>
       <scope>compile</scope>
     </dependency>
     <dependency>
       <groupId>ch.qos.logback</groupId>
       <artifactId>logback-classic</artifactId>
-      <version>1.2.3</version>
+      <version>1.4.5</version>
     </dependency>
```

Comment:
Looking at https://stackoverflow.com/questions/9717518/java-lang-nosuchmethoderror-ch-qos-logback-classic-loggercontext-getbithtime, I wonder if we need to explicit add `logback-core` with the same `logback-classic` version. @myurtoglu

Reply:
Yeah, along the same lines, doing some Googling, I saw that the missing method [was introduced in version 1.30](https://logback.qos.ch/apidocs/ch/qos/logback/core/util/EnvUtil.html#logbackVersion()) but [we also have another dependency that still uses 1.2.11](https://github.com/googleapis/java-logging-logback/blob/883b22c051f83598d3732d6a59972c6b4e9765fb/pom.xml#L17).

Reply:
> Looking at https://stackoverflow.com/questions/9717518/java-lang-nosuchmethoderror-ch-qos-logback-classic-loggercontext-getbithtime, I wonder if we need to explicit add `logback-core` with the same `logback-classic` version. @myurtoglu

And this I don't know, [but classic seems to include core anyway](https://github.com/qos-ch/logback/blob/bba7a908c87a4a483860f6d49695ec2348c56e49/logback-classic/pom.xml#L26).

Reply:
Resolving conversation to unblock this change.